module-new/src/components/switch/switch.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { expect } from '@storybook/jest';
+import { Meta, StoryObj } from '@storybook/react';
+import { within } from '@storybook/testing-library';
+import * as React from 'react';
+
+import { Switch } from './switch.component';
+
+export default {
+  title: 'Controls/Switch',
+  component: Switch,
+} as Meta<typeof Switch>;
+
+type Story = StoryObj<typeof Switch>;
+
+export const Default: Story = {
+  args: {
+    label: 'Airplane mode',
+  },
+  play: async ({ canvasElement }) => {
```

Comment:
is it necessary to separately test that the form bindings (via the `bind` prop) are also hooked up correctly?